tls/s2n_cipher_suites.c

```diff
@@ -1037,12 +1037,14 @@ int s2n_cipher_suites_init(void)
         }
     }
 
+    if (!getenv("S2N_SHARE_LIBCRYPTO")) {
```

Comment:
I think I would prefer a function in the public API that's well documented in what it does rather than an environment variable.

Reply:
Ok, so something like `s2n_share_libcrypto()` that must be called before `s2n_init`?

Reply:
yep something like that. but maybe we call it something like `s2n_libcrypto_disable_init()`?